### ISSUE TITLE
New GreyHack releasing in a day or two. Regex is coming.

### ISSUE THREAD

**Issue by Olipro** [ayecue/greybel-vs#117]
A few of the string functions have been replaced, in fact, `replace` has been replaced. `split`, `is_match` `replace_regex` and `matches` too.

They now use the C# Regex engine. I guess Javascript's will be close enough.

There's an optional additional param that gets mapped to the C# type:

```C#
if (option == "i")
{
	return RegexOptions.IgnoreCase;
}
if (option == "m")
{
	return RegexOptions.Multiline;
}
if (option == "s")
{
	return RegexOptions.Singleline;
}
if (option == "n")
{
	return RegexOptions.ExplicitCapture;
}
if (!(option == "x"))
{
	return RegexOptions.None;
}
return RegexOptions.IgnorePatternWhitespace; //default
```

**Comment by ayecue** [comment]
Intrinsics, Syntax and Hover texts should be available with 1.9.5 - https://github.com/ayecue/greybel-vs/pull/118.

**Comment by Olipro** [comment]
Superb!
